src/components/NotificationGrid.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react'
+import { Grid } from '@mui/material'
+
+export function NotificationGrid({
```

Comment:
`NotificationGrid` seems a strange name for this.

Reply:
This component renders a bulma "notification"